### PR TITLE
[RW-9731][risk=no]   Update documentation links about Cromwell

### PR DESCRIPTION
**Pull request by NehaBroad**
ezgif com-video-to-gif (1)

Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/components/runtime-configuration-panel/styles.tsx [hunk 1]
```diff
@@ -142,4 +142,11 @@ export const styles = reactStyles({
   diskLabel: {
     fontWeight: 500,
   },
+  disabledCloudProfile: {
```
Comment (evrii) [inline ui/src/app/components/runtime-configuration-panel/styles.tsx:145]: I just noticed that cromwell is using the runtime-configuration-panel styles. I wonder if we want to either move this file to somewhere common or split out the app styles into its own file. I do not believe that this has to be done now.

**Review by evrii — APPROVED** [review]
Looks good to me. Ran locally and worked as expected.

File: ui/src/app/components/runtime-configuration-panel/styles.tsx (hunk 1 above)
Comment (NehaBroad) [reply ui/src/app/components/runtime-configuration-panel/styles.tsx:145]: I agree thats separate PR, but i like the idea of moving this file to a common location

**Comment by NehaBroad** [comment]
The failing tests are flaky and not related to the code changed

**Pull request merged**